SA/design_concept/design_concept.typ

```diff
@@ -0,0 +1,17 @@
+#import "../style.typ": *

```

Comment:
Do we want this document to have the same setup / styling as the main document? I would like it if it was simpler and not so formal.

Reply:
good for now